kinetic/distribution.yaml

```diff
@@ -13215,6 +13215,22 @@ repositories:
       url: https://github.com/ros-visualization/visualization_tutorials.git
       version: kinetic-devel
     status: maintained
+  visualstates:
+    doc:
+      type: git
+      url: https://github.com/JdeRobot/VisualStates.git
+      version: 0.1.1
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/JdeRobot/VisualStates-release.git
+      version: 0.1.1-0
+    source:
+      test_pull_requests: true
```

Comment:
Just checking: Have you followed the [instructions prompted by bloom](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.

Reply:
I will remove test_pull_requests line, since we do not have any test currently.